README.md

```diff
@@ -0,0 +1,6 @@
+# Glimpse.ApplicationInsights
+
+####Application Insights and Glimpse integration
+
+Application Insights provides developers a tool for monitoring performance, availability, and usage of their applications. If a developer wants to view this data today, the only way is to use the Azure portal. With the Glimpse and Application Insight integration we can offer users an immediate display of their data over the F5 view of their application. This allows them direct, immediate access to their data. The three levels of experience we want to achieve are: **telemetry at your fingertips** by providing data collected through Application Insights for individual instances in the Glimpse viewer, **infinite history** by extending the ‘History’ tab in Glimpse, and **see production data during development** by adding a button to the Glimpse viewer linking corresponding production data from Application Insights. Telemetry is a cornerstone of modern applications. We believe integrating Application Insights and Glimpse will help developers get more insight into their applications. This will lead to more successful applications being produced.
```

Comment:
Make the first "Application Insights" to be a link to Application Insights landing page